src/bin/nps.js

```diff
@@ -87,6 +88,11 @@ function getPSConfigFilepath() {
 }
 
 function onInit() {
+  if (getPSConfigFilepath()) {
+    if (!keyInYN(colors.yellow(`Do you want to overwrite your existing config file?`))) {
```

Comment:
The library adds that for us:
<img width="485" alt="screen shot 2017-01-10 at 00 41 48" src="https://cloud.githubusercontent.com/assets/36491/21795065/9ba92cc0-d6cd-11e6-8b75-b7ec84acd26a.png">